common/esxi_get_model.yml

```diff
@@ -24,7 +24,7 @@
     esxi_cpu_vendor: "{{ esxi_host_property_results.ansible_facts.hardware.cpuPkg[0].vendor | default('Unknown') }}"
 
 - name: "Get ESXi server CPU code name"
-  when: esxi_cpu_vendor | lower == 'intel'
```

Comment:
If 'lower' is removed from where it's called, then it need to be added when it's defined to make sure the value is indeed in lower case.

Reply:
Add the filter to its definition.